package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh

```diff
@@ -291,7 +291,6 @@ mac80211_hostapd_setup_base() {
 		esac
 		[ -n "$op_class" ] && append base_cfg "op_class=$op_class" "$N"
 	}
-	[ "$hwmode" = "a" ] || enable_ac=0
```

Comment:
Doesn't this enable 802.11ac/QAM256 unconditionally?

Reply:
if i'm not totally fked up today removing this line means enable VHT unconditionally with VHT mode selected. HT should set enable_ac to 0

Reply:
it's pretty clean compared to the previous patches, I just checked and it seems OK actually.... just surprised this is all it took because I remember how much of a claustrophobic the first patch was when it came to changes to mac80211.sh